hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/HoodieSparkUtils.scala

```diff
@@ -58,6 +58,7 @@ private[hudi] trait SparkVersionsSupport {
   def gteqSpark3_2_1: Boolean = getSparkVersion >= "3.2.1"
   def gteqSpark3_2_2: Boolean = getSparkVersion >= "3.2.2"
   def gteqSpark3_3: Boolean = getSparkVersion >= "3.3"
+  def gteqSpark3_3_2: Boolean = getSparkVersion >= "3.3.2"
```

Comment:
Is this based on compile-time Spark version or runtime Spark version?

Reply:
Should be runtime, I think.